### ISSUE TITLE
Having Trouble Staying Alive!

### ISSUE THREAD

**Issue by MrKnitster** [cambragol/advent-of-the-trinity#40]
I have to admit, I am really having trouble keeping everyone alive. I am gaining and losing avatarhood in valor and honor, etc., just by dying too often. I have not even dared to explore wilderness or dungeons, because it's instant death. Not sure whether I'm approaching it wrong or if other people are experiencing the same thing.

I look forward to getting to a stage when I can start to make progress on the actual quests...

**Comment by Fenyx4** [comment]
I think that I may have gotten past the dying all the time hump.

Let me know if you want some hints.

**Comment by cambragol** [comment]
Of course, we want there to be a challenge, a tough one, but not enough to make it not fun. I'd also expect a 'modern' gamer to drop out early on. You need the patience and persistence of an old school avatar/gamer to make it through.

Thinking back, I know that we all died DOZENS if not hundreds of times in our original play throughs. Is Trinity keeping form to this? Because it should I think.

If Fenyx4 has gone over the hump, that is good. Though, I would like to know if there was any special strategy, or did you just fight it out, gain experience, and repeat?

I would like the strategy to be something normalish, and not something 'gamey'. If it has to be 'gamey' that is no good.

**Comment by MrKnitster** [comment]
Hints welcome!

**Comment by Fenyx4** [comment]
I did get a bit gamey.

My character is a mage and I started out trying to go solo as that is a good strategy in the base game. 
That didn't work. Between more monsters, more damaging monsters and ranged weapons not hitting close I was dying a lot.
So I grabbed Iolo and Jaana and things improved. Got Mandrake and Nightshade. I had some limited luck with a strategy of everyone having slings and having two staves to swap to when things got close. Got up to levels 5, 4 and 4.

But I was still dying the groups of mages and was squishy enough I feared doing much exploration... I was having problems getting up enough gold to get a significant amount of reagents. Even though I had found Hargrove which was a good source of reagents. And I was having problems getting enough gold to get some keys. 

Soo... After a failed dip in Dewmire I decided to pull out all stops. Don't read below unless you want to know how to abuse the game.

TL;DR I cheesed my way into Dungeons to get orbs.

Mixed up a few blinks, zdowns and xits. Walked to Despise. Monsters poured out of Despise and it was hard to get in. (Which is an interesting addition). Once I was in I zdowned down to level 3, grabbed the 4 chests on that level and went down the ladder. On level 4 I grabbed the orb to increase my main character's dex. Zdowned down to level 6. Made use of the healing fountain, grabbed all the chests and xited. Re-entered and rinsed repeated.

Got killed by reapers catching me once so I also mixed up some negates to protect me from sleep.

Whenever I had to go get more reagents I began blinking to the entrance of Despise to avoid the waves of monsters.

Got my main character up to 50 dex (once you hit 40 dex you hit 100% of the time. Major boost to damage.)

Blinked to Shame and did similar but with zdowning to level 6 to get the orbs and raise my main characters int and dex.

Went back to Despise and got Iolo and Jaana's dex to 50. Can't get them the Shame orbs yet without killing them.

The dex stat makes me much deadlier in combat. Gold let me get reagents for spells which makes combat much more survivable.
Gold also let me get a bunch of keys and get a Magic Bow. Keys are finally letting me explore towns again.

**Comment by Fenyx4** [comment]
P.S. fights still aren't a cake walk but I don't feel like I am courting death walking between Yew and the Abbey. 😃

**Comment by cambragol** [comment]
Hmm, yes just a bit gamey!

I guess the question is, could you have made it without gaming it?

Also, ideally we wouldn't be able to get over a 'hump' but rather the progression through levels and battles, gaining gold and experience etc., would be a constant struggle/challenge that persists as long as possible. When that fades, the game, any game, becomes less interesting to most people. In Ultima IV we do have a second part of the game, which is the questing, and that can extend the gameplay of course.

So I/we need to be sure that the balance is good and tough, doable, and stretched out, but not too hard, overly frustrating etc.

I will need your honest feedback on that, so I can tweak things as best as possible.

**Comment by Fenyx4** [comment]
I need to get some more playtime in to see just how over the hump I am. I do still only have a staff for a melee weapon after all...

I may have been able to do it. But I was getting frustrated cuz, even though I enjoy the increased difficulty, I wanted to get past the old content and get into new content!

What class do you usually play cambragol? I may have hamstrung myself with going mage and having my best melee weapon for a long, long time being a staff. Perhaps I shoulda tried more combat spells like magic missile and fireball...

If you would like I could start over.

**Comment by cambragol** [comment]
I think it is fine to head onto the new content. By all means head on!

We'll just need to be sure at some point about the balance with mobs/xp etc. before we finish. General play of any form will continue to give us insights into that, and if necessary, we can do some rookie run-throughs later, as tests.

I never noticed classes when playing, only stayed true to the virtue choices I made in the beginning, which usually led me to Britain or Minoc, I believe. Though, it has been some time since I did a true play through.

Certainly though, different classes will make a big difference. One thing I was looking at a lot was trying to make the spells in the game more 'relevant'. I've read many a review or journal where people complained of the uselessness of spells, and how the game can be done without them. One of my hopes was that increased difficulty with combat would make certain spells more useful. As an example the Poison state cheat is gone, so to stop Zorns, Balrons and Reapers etc., you will need to cast Negate more. Things like that.

**Comment by MrKnitster** [comment]
I put together a party of about 3 people, magic axe for one (best in my opinion, as it involves no switching), and bow/sling and sword/staff for the other. I can usually stay alive as long as I don't journey too long without a trip to LB to top up on hit points...

**Comment by cambragol** [comment]
That sounds...good?

**Comment by MrKnitster** [comment]
Good-ish. It's still a grind.
